### ISSUE TITLE
useMenuTrigger example warns about "An aria-label or aria-labelledby prop is required for accessibility."

### ISSUE THREAD

**Issue by LFDanLu** [adobe/react-spectrum#2610]
# 🐛 Bug Report 

When you open the useMenuTrigger example in the[ useMenuTrigger docs](https://react-spectrum.adobe.com/react-aria/useMenuTrigger.html#example), the browser console warns about missing aria-label/aria-labelledby props. Figure out why this is happening (useMenu hook warning? But the menu has an aria-labelledby?)

## 🤔 Expected Behavior

No warning in the browser console

## 😯 Current Behavior

Browser console warns about a missing aria-label/aria-labelledby prop

## 💁 Possible Solution

Maybe as simple as passing an aria-label/labelledby to useMenu in the example? Or perhaps the hook need a fix?

## 🧢 Your Company/Team

RSP

**Comment by frdclav** [comment]
Working on this one, it looks like the aria-labelledby is getting passed into domProps. I made a small change to pass the value directly into the MenuPopup component instead of into domProps. https://github.com/adobe/react-spectrum/pull/2642 Is this the right way to go about this?

**Comment by snowystinger** [comment]
Closed by PR mentioned
